rebar.config

```diff
@@ -1,106 +1,125 @@
-%%% -*-mode:erlang;coding:utf-8;tabidth,:4;c-basic-offset:4;indent-tabs-mode:()-*-
-%%% ex: set ft=erlang fenc=utf-8 sts=4 ts=4 sw=4 et:
-%%%
+%% == Erlang Compiler ==
 
-%%% Require OTP 19.1 at a bare minimum
-{minimum_otp_vsn, "19.1"}.
+{minimum_otp_vsn, "22"}.
 
-%% Plugins
-{plugins, [rebar3_hex]}.
-{shell, [
-    {apps, [lager, sync, gen_rpc]}
-]}.
-
-{erl_opts, [debug_info,
-    {warn_format, 1},
-    bin_opt_info,
-    inline_list_funcs,
-    warnings_as_errors,
-    report_warnings,
-    warn_untyped_record,
+{erl_opts, [
+    debug_info,
     warn_export_vars,
-    warn_export_all,
-    warn_shadow_vars,
-    warn_obsolete_guard,
     warn_unused_import,
-    warn_deprecated_function,
-    warn_unused_vars,
-    warn_unused_function,
-    warn_bif_clash,
-    warn_unused_record,
-    warn_exported_vars
+    warnings_as_errors,
+    {i, "src"}
 ]}.
 
+%% == Dependencies and plugins ==
+
 {deps, [
-    {hut, "~> 1.2"},
-    {ssl_verify_fun, "~> 1.1"}
+    {hut, "1.3.0"},
+    {ssl_verify_hostname, "1.0.6"}
```

Comment:
🤔 

Why the change from `ssl_verify_fun` to `ssl_verify_hostname`?

Reply:
`ssl_verify_hostname`'s GitHub link in Hex points to `ssl_verify_fun`'s repo 😵‍💫

Reply:
On Hex: the last version of `ssl_verify_hostname` is `1.0.6`, the first version of `ssl_verify_fun` is `1.0.8`, so I'm gonna throw a guess that the library was renamed.

If my hypothesis is true, it implies we're actually importing an older version of `ssl_verify_fun` (pre-rename) and should therefore keep using `ssl_verify_fun`.

Reply:
The code uses `ssl_verify_hostname`, so I'm not actually sure what happened there. We might want to move to `ssl_verify_fun`, sure. `ssl_verify_hostname` is `ssl_verify_fun` renamed. Not sure about the versions, since I didn't check. Will do, later.

Reply:
`ssl_verify_fun` includes a module named `ssl_verify_hostname`, that's what likely misguided you

Reply:
👍 let me check this.

Reply:
```
src/driver/gen_rpc_driver_ssl.erl
Line 181 Column 49: Unknown function ssl_verify_hostname:verify_cert_hostname/2
Line 256 Column 33: Unknown function ssl_verify_hostname:verify_fun/3
```

(will dig deeper)

Reply:
I think this mess is now untangled with 57adedb. `test` was pulling in a different lib./reference and screwing the analysis.

Reply:
😵 

👍